linux-login-kubic-20180201.json

```diff
@@ -24,7 +24,7 @@
   ],
   "tags": [
     "ENV-DISTRI-KUBIC",

```

Comment:
what about this tag ? does it still make sense ?

Reply:
I guess so, there are flavors `Kubic-DVD` and `MicroOS-Image-Kubic-kubeadm` in O3